src/contents/articles/cold-drinks-were-the-remedy.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Cold Drinks Were the Remedy
+author: msanatan
+date: 2015-06-29
+template: article.jade
+---
+
+What do Code and Drinks have in common? Not that much thankfully (imagine the bugs). Since we had an event for [coding](/articles/public-coding-roundup.html) recently, this time we got about the drinking part!
```

Comment:
Will change, error in my thought-to-text mapping here. Will change as follows:

'What do Code and Drinks have in common? Namespace events! Thankfully not together (imagine the bugs). Since we had an event for [coding](/articles/public-coding-roundup.html) recently, this time we got about the drinking part!'